### ISSUE TITLE
Settings - Views, all items disabled

### ISSUE THREAD

**Issue by pcrov** [jellyfin/jellycon#206]
**Describe the bug**
JellyCon addon settings > Views has only a list of disabled items:

- Movies view ID
- TV Shows view ID
- Seasons view ID
- Episodes view ID
- Collections view ID

These are all grayed out and cannot be interacted with.

**To Reproduce**

1. Go to the JellyCon addon settings
2. Click on Views
3. Voilà

**Expected behavior**
I'm not entirely sure, I just started using this. My guess would be a way to select or input default views for each item, which would be most welcome.

**Logs**
I could find no relevant errors in the log.

**Screenshots**
![Views](https://user-images.githubusercontent.com/8586747/184338877-f374e885-60a3-4e3a-9daf-e18370038130.png)

**System (please complete the following information):**
 - OS: Windows
 - Jellyfin Version: 10.8.3
 - Kodi Version: 19.4
 - Addon Version: 0.5.5+py3

**Additional context**
https://github.com/jellyfin/jellycon/blob/11705d46f26e1315f8c1d3dbad2eaf9143d1aca6/resources/settings.xml#L132-L140

Maybe I'm just not understanding what this is supposed to be.

**Comment by mcarlton00** [comment]
I _believe_ this is meant to be more informational than an actual editable setting, but it's inherited behavior that I didn't actually know about until a few weeks ago and I'm still somewhat mixed on it.

When you're viewing lists of items in the library, you can press Left and do things like filter items or change the ordering.  One of those options is "View Type".  You can change that and it changes how the items appear on screen.  But when using the addon, it only changes items for that particular folder.  So if you change the view type of Season 01 of an item, Season 02 will still be the default.  To combat this, the previous dev added an option to the context menu called "Set default menu".  This saves the current view type for that content type and will make it take effect for all other items of that same type.

It saves those views to this section of the settings.  The problem is that they're not exactly human readable in their current form.  This is with Movies set to "InfoWall" and Collections set to "Fanart", for example.
![2022-08-12-082225_1821x898_scrot](https://user-images.githubusercontent.com/17029228/184353743-74319527-9d58-48cf-9a05-a25c182e6761.png)

**Comment by mcarlton00** [comment]
I agree.  It's a rather clunky way of handling this, and that's why I only realized how it works a few weeks ago even though I've been maintaining this addon since July of 2020.

In theory, it should be possible to manually map all the ID numbers to their names so they can be displayed and selected here.  I'm not sure what that code would actually look like yet though.

**Comment by mcarlton00** [comment]
So this is actually going to be more complicated than I thought.  A short summary

* As far as I can tell, there's no way for an addon to query what view types are available, and it seems to be highly dependent on the skin and media type being shown at that moment
* In theory, it should be possible to manually map each ID to it's name.  But!
* Skins and other addons can add new view types.  The settings menu options can't really be updated dynamically, and as mentioned there's no way to look those up.

It's looking like this may be a case of needing a better UX and documentation for how to set those, and not being able to use the settings menu to do it "nicely".

**Comment by matt77303** [comment]
Is there any upadtes on this? 
id like to set a global viewtype for episodes but it only changes the one that im looking at. 

Thank you 

Matt

**Comment by coreflake1** [comment]
@mcarlton00 

It would be great if you could just enable the fields. 
If setting manually in settings.xml (in userdata), it works perfectly as-is ! 
for example if I edit the settings.xml like this : 
`<setting id="view-movies" default="true">54</setting>`

It will always use viewtype 54 as default for all. 

The viewtype is really easy to find (even with custom skins), just go to the addon folder (not userdata) and in addon.xml you set "debug=true". 
It will give you an overlay displaying what viewtype is selected. 

The Youtube addon has implemented this in exactly the same way and it works flawless ! 
Also with this addon it works flawless, it's just a shame that I need to edit the settings.xml manually.
